app/services/physical_holdings_markup_builder.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: false
 
 class PhysicalHoldingsMarkupBuilder < HoldingRequestsBuilder
+  include ApplicationHelper # needed for find_it_location method
```

Comment:
I don't think this comment is needed, is it? I think it would be enough adding it as a comment in your pr.

Reply:
Done